### PR TITLE
Copy files from openshift_master_generated_config_dir instead using hardlinks

### LINKED ISSUE #7075: Errors, if `openshift_master_generated_config_dir` is not local.
#### Description

Getting errors on the installation, if `openshift_master_generated_config_dir` is not local. Because hard links work only on the same device.

See here: https://github.com/openshift/openshift-ansible/blob/dffd34a1b4088d32ccbf69ce58ab930575146e8f/roles/openshift_master_certificates/tasks/main.yml#L105

##### Version

```
ansible 2.4.2.0
openshift-ansible-3.7.29-1-2-gdffd34a1b
```

##### Steps To Reproduce
1. Mount a device or fs under `/etc/origin/generated-configs`
2. run the Ansible Playbook

##### Expected Results
Just works.

##### Observed Results
Ansible errored.

```
2018-02-08 19:36:29,088 p=2289 u=jkr |  failed: [openshift-master2 -> 10.124.0.10] (item=admin.crt) => {
    "changed": false,
    "gid": 0,
    "group": "root",
    "item": "admin.crt",
    "mode": "0644",
    "owner": "root",
    "path": "/etc/origin/generated-configs/master-ip-10-124-1-11.eu-central-1.compute.internal/admin.crt",
    "secontext": "system_u:object_r:nfs_t:s0",
    "size": 1119,
    "state": "hard",
    "uid": 0
}

MSG:

Error while replacing: [Errno 18] Ungültiger Link über Gerätegrenzen hinweg

2018-02-08 19:36:29,252 p=2289 u=jkr |  failed: [openshift-master3 -> 10.124.0.10] (item=admin.crt) => {
    "changed": false,
    "gid": 0,
    "group": "root",
    "item": "admin.crt",
    "mode": "0644",
    "owner": "root",
    "path": "/etc/origin/generated-configs/master-ip-10-124-2-64.eu-central-1.compute.internal/admin.crt",
    "secontext": "system_u:object_r:nfs_t:s0",
    "size": 1119,
    "state": "hard",
    "uid": 0
}

MSG:

Error while replacing: [Errno 18] Ungültiger Link über Gerätegrenzen hinweg
```

##### Additional Information

Why we are using NFS for this path?

Since we are using AWS and a AWS dynamic inventory, the first master (groups.oo_first_master.0 or openshift_ca_host) isn't constant. (And we don't want have cattles even for our masters!) Thats we want every master can be the ca_host. (Mounting the `openshift_master_generated_config_dir` directory to an Shared Medium is maybe the workaround)

### PR DESCRIPTION
**Pull request by jkroepke**
Fixes #7075

Tested in a release-3.7 branch on a existing 3.7 cluster.

### CODE REVIEW

**Comment by jkroepke** [comment]
anyone?

**Comment by jkroepke** [comment]
What someone merge the small PR?

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by jkroepke** [comment]
Hi,

when merged, can you backport it into 3.9?

File: roles/openshift_master_certificates/tasks/main.yml [hunk 1]
```diff
@@ -88,11 +88,11 @@
   delegate_to: "{{ openshift_ca_host }}"
   run_once: true
 
-- file:
+- copy:
     src: "/etc/origin/master/{{ item }}"
     dest: "{{ openshift_master_generated_config_dir }}/{{ item }}"
     state: hard
```
Comment (michaelgugino) [inline roles/openshift_master_certificates/tasks/main.yml]: Remove state: hard

**Comment by michaelgugino** [comment]
@jkroepke please squash the commits into one.

**Comment by jkroepke** [comment]
Done.

**Comment by michaelgugino** [comment]
bot, retest this please

**Comment by jkroepke** [comment]
Any chance to get this into 3.9 ?

/cherrypick release-3.9

**Comment by michaelgugino** [comment]
@jkroepke Looks like we'll need to pick this by hand.  Please create a PR against 3.9 and we'll get that merged.  It's helpful to use the command git cherry-pick -x <commit id>

**Comment by jkroepke** [comment]
Done. Thanks.

**Pull request merged**